### ISSUE TITLE
Sidenav demo accessibility issues

### ISSUE THREAD

**Issue by tinayuangao** [angular/components#7186]
#### Bug, feature request, or proposal:
Bug

#### What is the expected behavior?
After clicking on "View example", the focus should move to sidenav menu icon.
After clicking on sidenav menu icon, the focus should move to sidenav links.
After clicking on "Close example", the focus should be removed from "Close example" button

#### What is the current behavior?
After clicking on "View exmaple", the focus stayed on "View example" button.
After clicking on sidenav menu icon, the focus stayed on sidenav menu icon.
After clicking on "Close example" button, the focus stayed on the "Close example" button

#### What are the steps to reproduce?

Providing a Plunker (or similar) is the *best* way to get the team to see your issue.
Plunker template: https://goo.gl/DlHd6U
https://material2-accessibility.firebaseapp.com/accessibility/sidenav

**Comment by antonyboom** [comment]
Is there any fix for this bug ?
